jujuclient/setmodels_test.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package jujuclient_test
+
+import (
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/jujuclient"
+	"github.com/juju/juju/testing"
+)
+
+type ControllerModelsSuite struct {
```

Comment:
Yes, I could but it felt to me that existing suite had more than what I needed in terms of controllers and models.